products/sle/main.pm

```diff
@@ -1140,6 +1145,24 @@ elsif (get_var("QA_TESTSET")) {
     }
     loadtest "qa_automation/" . get_var("QA_TESTSET") . ".pm";
 }
+elsif (get_var("VIRT_AUTOTEST")) {
+    #loadtest "virt_autotest/common/init_pxe_install.pm";
```

Comment:
Don't add disabled code as nobody but you will understand why this is not active. Better leave it out and keep in your local git repository, either in `git stash` or a temporary "WIP"-commit